package.json

```diff
@@ -1,12 +1,16 @@
 {
-  "name": "gatsby-source-rss",
-  "version": "1.0.0",
+  "name": "gatsby-source-rss-fork",
```

Comment:
Wanna change this back to `gatsby-source-rss` so that it's the "official", unforked version?

Reply:
Of course.